open-craft/opencraft
# [BB-2558] Temporarily set default `FORUM_MONGO_AUTH_MECH` to `MONGODB-CR`

This is a temporary change that sets the default authentication mechanism to MONGODB-CR. It will be reverted once we upgrade to [SCRAM](https://docs.mongodb.com/manual/core/security-scram/).

This option is ignored in Ironwood, so it affects only Juniper instances, which have it set to SCRAM by default.

This does not include testing the fix itself, because we're already using it, cf. [relevant discussion](https://discuss.openedx.org/t/discussion-forum-errors-in-latest-master-with-mongo-3-2/1843/7) and [example instance](https://manage.opencraft.com/instance/19980/edx-appserver/13102/).

## Testing instructions
1. Check that [the test AppServer](https://stage.manage.opencraft.com/instance/5450/edx-appserver/2077/) contains `FORUM_MONGO_AUTH_MECH: :mongodb_cr` in its `Configuration`.

## Manual testing instructions
1. Check the branch out on stage, [create an instance](https://ocim.opencraft.com/en/latest/provisioning-sandboxes/#manual-provisioning) and spawn new AppServer.
1. In Ocim go to `AppServer -> Configuration` and confirm that `FORUM_MONGO_AUTH_MECH: :mongodb_cr` is present there.
